src/service/url-replacements-impl.js

```diff
@@ -753,6 +193,7 @@ export function installUrlReplacementsServiceForDoc(ampdoc) {
  * @param {*} varSource
  */
 export function installUrlReplacementsForEmbed(ampdoc, embedWin, varSource) {
+  installVarSubstitutionForEmbed(ampdoc, embedWin, varSource);
   installServiceInEmbedScope(embedWin, 'url-replace',
-      new UrlReplacements(ampdoc, varSource));
+      new UrlReplacements(ampdoc));
```

Comment:
I am installing it for the embed scope, I think that should be enough no?

cc/ @avimehta